### PR TITLE
Detail

### PR DESCRIPTION
**Pull request by MohamedSbaih**
Add the first section

### CODE REVIEW

**Review by MostafaBalousha123 — APPROVED** [review]

**Review by Mo3en-ps — APPROVED** [review]

File: details/details.html [hunk 1]
```diff
@@ -4,8 +4,25 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./details.css/" />
-    <title>Document</title>
+    <link rel="stylesheet" href="style/style.css" />
```
Comment (sara219) [inline details/details.html:7]: fix the link of the style file 
```suggestion
    <link rel="stylesheet" href="./details.css" />
```

**Review by sara219 — CHANGES_REQUESTED** [review]
to merge this pull request you should complete the whole page 
you did a great thing by pushing it section by section, but it's not ready to review yet till you're done with the page

File: details/details.html (hunk 1 above)
Comment (MohamedSbaih) [reply details/details.html]: Modified

**Review by MohamedSbaih — COMMENTED** [review]
Modified

**Review by techhana2014 — APPROVED** [review]

**Review by sara219 — DISMISSED** [review]

**Review by Mo3en-ps — APPROVED** [review]

**Review by MostafaBalousha123 — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**